### PR TITLE
Adding a red theme

### PR DESCRIPTION
**Pull request by m1ga**
Make it red:

theme

### CODE REVIEW

File: app/platform/android/alloy_generated [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This directory is generated from the "/app/platform" and "/app/theme/<name>/platform" directories.
+
+Do not modify any files in this directory. Your changes will be lost on next build.
+
+Please make sure "/platform" is added to your version control's ignore list (i.e. .gitignore).
```
Comment (hansemannn) [inline app/platform/android/alloy_generated]: Wondering how this does generated in `app/platform`, not `platform`. Can you add the .gitignore specs here?

File: tiapp.xml [hunk 2]
```diff
@@ -23,8 +23,11 @@
 			</dict>
 		</plist>
 	</ios>
-	<android xmlns:android="http://schemas.android.com/apk/res/android" />
-	<modules />
+	<android xmlns:android="http://schemas.android.com/apk/res/android">
+        <manifest android:versionCode="1" android:versionName="2.0.0">
```
Comment (hansemannn) [inline tiapp.xml]: Please use 1.1.0 instead

File: app/platform/android/alloy_generated (hunk 1 above)
Comment (m1ga) [inline app/platform/android/alloy_generated]: my bad. I've copied the wrong folder and forgot to remove that file! Didn't check the PR

File: tiapp.xml [hunk 3]
```diff
@@ -23,8 +23,11 @@
...
 		</plist>
 	</ios>
-	<android xmlns:android="http://schemas.android.com/apk/res/android" />
-	<modules />
+	<android xmlns:android="http://schemas.android.com/apk/res/android">
+        <manifest android:versionCode="1" android:versionName="1.1.0">
+            <application android:theme="@style/kitchensink" android:hardwareAccelerated="true" android:largeHeap="true"/>
+        </manifest>
+    </android>
 	<deployment-targets>
```
Comment (hansemannn) [inline tiapp.xml]: Indentation mixed up. Please use the 4-spaces-indentation like the rest of the tiapp.xml. We can merge after that, thx!

**Comment by hansemannn** [comment]
Approved! :rocket:

**Pull request merged**